src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private String model;
+    private int bucketVolume;
+
     public Excavator() {
     }
 
+    public Excavator(String name, String color, String model, int bucketVolume) {
+        setName(name);
+        setColor(color);
```

Comment:
```suggestion
        super(color, name);
```
constructor is usually used

Reply:
> constructor is usually used

done